xgress_udp/address.go

```diff
@@ -29,27 +28,10 @@ var supportedNetworks = map[string]string{
 	"unixgram": "",
 }
 
-type packetAddress struct {
-	network string
-	address string
-}
-
-func (pa *packetAddress) String() string {
-	return fmt.Sprintf("packetAddress{network=%v, addr=%v}", pa.network, pa.address)
-}
-
-func (pa *packetAddress) Network() string {
-	return pa.network
-}
-
-func (pa *packetAddress) Address() string {
-	return pa.address
-}
-
-func parseAddress(s string) (*packetAddress, error) {
+func Parse(s string) (*PacketAddress, error) {
 	tokens := strings.Split(s, ":")
 	if len(tokens) < 2 {
-		return nil, errors.New("invalid format")
```

Comment:
Is there a difference between errors.New and fmt.Errorf if there are no msg parameters?

Reply:
Haven't measured it. But I tend to like the consistency of just always using `fmt.Errorf`.